### PR TITLE
Update cubedash processing script inline with eks prod stack updates

### PR DESCRIPTION
**Pull request by santoshamohan**
## Updates in this PR

**requirements-test.txt** 
- Pin `datacube` version to `1.7+20.ga81eb035`

**.travis.yml**
 - Install `coveralls` package to enable coverage stats online
 - Show coverage stats only after all the checks are passed
 - Do not increment `docker` tag minor version. `datakube-apps` service `yaml` files use `SemVer` standards
 
**Dockerfile**
 - Pin to latest `datacube-core` 1.7
 - pip install `gevent`
 - Copy `nci` db update bash script and `aws` db update bash script
 - Fix "Critical Worker Timeout" issue when running `cubedash` app, by increasing timeout to `121` and run `gunicorn` with 3 workers
 
**Makefile**
- Fix multiple code coverage runs

**deployment/load-nci-dump.bash**
- Refactor the bash script
- Configure `psql` environment variables as input argument
- Remove any `pgdump` files (if exists)
- Remove references to `psql_args` since we are pulling the `psql` credentials from secrets
- Save the `datacube` db information on `datacube.conf` file
- Once extents generation is completed, publish `cubedash` database update to `SNS` topic
- Finally clean old databases older than 1 day

**deployment/load-rds-db-dump.bash**
- Updates are similar to that of `deployment/load-nci-dump.bash` except no db dump and restore is performed

### CODE REVIEW

File: deployment/load-nci-dump.bash [hunk 1]
```diff
@@ -94,38 +97,47 @@ psql ${psql_args} "${dbname}" -X -c 'copy (select name from agdc.dataset_type or
 echo "
 [datacube]
 db_database: ${dbname}
+db_hostname: db-prod-eks-datacube-default.cfeq4wxgcaui.ap-southeast-2.rds.amazonaws.com
```
Comment (tom-butler) [inline deployment/load-nci-dump.bash]: DB config should be passed as a variable, not hard coded

**Review by tom-butler — CHANGES_REQUESTED** [review]

File: deployment/load-nci-dump.bash (hunk 1 above)
Comment (santoshamohan) [reply deployment/load-nci-dump.bash]: Updated to pass db config as variables

**Review by jeremyh — CHANGES_REQUESTED** [review]
Re:our discussion -- rerequest when ready for review again

**Review by santoshamohan — COMMENTED** [review]
Ready for re-review

**Review by jeremyh — DISMISSED** [review]
:+1: 

Thanks Santosh

File: deployment/load-nci-dump.bash [hunk 2]
```diff
@@ -1,14 +1,22 @@
...
 
+# Remove any previous pgdump files before processing cubedash job
+rm -rf /data/nci/*-datacube.pgdump
+
 # Optional first argument is day to load (eg. "yesterday")
 dump_id="$(date "-d${1:-today}" +%Y%m%d)"
-psql_args="-h db-prod-eks-datacube-default.cfeq4wxgcaui.ap-southeast-2.rds.amazonaws.com -U superuser"
+export PGUSER="superuser"
+export PGHOST="db-prod-eks-datacube-default.cfeq4wxgcaui.ap-southeast-2.rds.amazonaws.com"
+export PGPORT=5432
```
Comment (tom-butler) [inline deployment/load-nci-dump.bash:17]: These should be pulled from env vars, not hard coded as env vars

**Review by tom-butler — DISMISSED** [review]

File: deployment/load-nci-dump.bash (hunk 2 above)
Comment (santoshamohan) [reply deployment/load-nci-dump.bash:17]: Updated

**Review by jeremyh — DISMISSED** [review]

**Comment by alexgleith** [comment]
Is this running? Can we either kill it or shift it into a better place? Ping @tom-butler

**Pull request closed without merging**